.gitattributes

```diff
@@ -0,0 +1 @@
+**/generated/** linguist-generated
```

Comment:
Mark generated files as linguist-generated to hide GitHub PR diff
display by default for generated schema file changes.

See GitHub Docs:
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github
https://github.com/github/linguist/blob/master/docs/overrides.md#generated-code